### PR TITLE
chore: replace ts-ignore with ts-expect-error

### PR DESCRIPTION
**Pull request by ferferga**

### CODE REVIEW

**Comment by Camc314** [comment]
Build failing?

**Review by ThibaultNocchi — APPROVED** [review]

File: components/__tests__/BlurhashCanvas.spec.ts [hunk 1]
```diff
@@ -14,45 +14,35 @@ beforeEach(() => {
...
-    // @ts-ignore
-    expect(wrapper.vm.$options.props.hash).toBeDefined();
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    expect(wrapper.vm.$options.props.hash.type).toBe(String);
+    // @ts-expect-error - props are not defined in the Vue type.
+    expect(wrapper?.vm?.$options?.props?.hash).toBeDefined();
+    // @ts-expect-error - props are not defined in the Vue type.
+    expect(wrapper?.vm?.$options?.props?.hash?.type).toBe(String);
   });
```
Comment (Camc314) [inline components/__tests__/BlurhashCanvas.spec.ts:21]:
```suggestion
```

File: components/__tests__/BlurhashCanvas.spec.ts [hunk 2]
```diff
@@ -14,45 +14,35 @@ beforeEach(() => {
...
 
-  test('has a width property of type Number', () => {
```
Comment (Camc314) [inline components/__tests__/BlurhashCanvas.spec.ts:25]: I don't think you meant to delete this line

Comment (ferferga) [reply components/__tests__/BlurhashCanvas.spec.ts:25]: Oops, good catch!

**Review by Camc314 — APPROVED** [review]

**Pull request merged**